### DISCUSSION TITLE
Streamline viewing Expo examples

### DISCUSSION THREAD

**Discussion by jeremy303** [nandorojo/moti#220] [General]
It appears there are great examples of Moti in action in the /examples directory, however for someone who just wants to see what is possible w/ Moti it can take a fair amount of effort to install Expo and it's dependencies and then get the examples project running.   I tried this and I'm still running into errors, I assume because the Moti lib must be built first.  (See below).

<details>
  <summary>Error when executing 'with-expo' example</summary>

```
iOS Bundling failed 4468ms
Unable to resolve module ../build/interactions from /Users/Jeremy/Developer/source/moti/packages/moti/interactions/index.js: 

None of these files exist:
  * ../../packages/moti/build/interactions(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)
  * ../../packages/moti/build/interactions/index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)
> 1 | export * from '../build/interactions'
    |                ^
  2 | //# sourceMappingURL=index.js.map
  3 | 

```

</details>
It would be great if a README could be added to the examples with a link to a deployed instance of the Expo app so a quick scan of a QR code could have the examples up and running; as well as instructions for running the Expo project locally.

Thanks!

**Comment by nandorojo** [comment]
yeah agreed…many of these example are just used for dev purposes / testing things. i’d be happy to approve a PR that adds good examples as snacks to the docs. there are already a few snacks in the docs, so you could just copy those
